### PR TITLE
Fix for install script to support dotnet 6

### PR DESCRIPTION
**Pull request by hsubramanianaks**
This PR will fix the issue https://github.com/Azure/Bridge-To-Kubernetes/issues/170. B2K CLI install scripts will now download dotnet 6 binaries if it is not present. Thanks for reviewing this. 

**WSL screenshot missing dotnet 6 binaries:**
**only dotnet 3 installed.**
image

image

image

**Windows git bash screenshot:**

image

image

### CODE REVIEW

**Review by Tatsinnit — DISMISSED** [review]
<3 Thank you so much, Looks good to me!!

**Review by elenavillamil — DISMISSED** [review]
Looks good! I am assuming this was manually validated in all 3 OS right? Let me know if you need help with mac.

**Comment by hsubramanianaks** [comment]
> Looks good! I am assuming this was manually validated in all 3 OS right? Let me know if you need help with mac.

Yes @elenavillamil can you run in Mac once , windows and WSL works fine. 
```curl -fsSL https://raw.githubusercontent.com/hsubramanianaks/Bridge-To-Kubernetes/fix-script-dotnet-6/scripts/install.sh | bash```

File: scripts/install.sh [hunk 1]
```diff
@@ -143,11 +150,23 @@ install_tool() {
     esac
 }
 
+install_dotnet_x64_for_arm() {
+    log INFO "downloading and installing dotnet x64 binaries in arm machines"
```
Comment (hsubramanianaks) [inline scripts/install.sh:154]: This is temporary method until we fix the build pipelines to produce arm64 binaries for bridge. Thank you.

Comment (harans87) [reply scripts/install.sh:154]: Works in M1 Mac with this temporary fix for arm machines.

curl -fsSL https://raw.githubusercontent.com/hsubramanianaks/Bridge-To-Kubernetes/fix-script-dotnet-6/scripts/install.sh | bash

Screen Shot 2023-02-23 at 2 17 15 AM


Screen Shot 2023-02-23 at 2 16 09 AM

Screen Shot 2023-02-23 at 2 18 07 AM

**Review by elenavillamil — APPROVED** [review]
LGMT. Lets make sure we add note to remove this on the task we open to update pipelines.

**Pull request merged**